charts/cluster-api-runtime-extensions-nutanix/addons/registry-mirror/distribution/values-template.yaml

```diff
@@ -0,0 +1,12 @@
+replicaCount: 2
```

Comment:
should we dynamically set the replicaCount = # of CP nodes? 
also set anti-affinities to schedule one statefulset per CP node to help during downtime, upgrade and single node cluster? https://github.com/mesosphere/charts/blob/master/staging/docker-registry/templates/statefulset.yaml  we need to enhance the chart.

Reply:
correction: affinity support is in chart. https://github.com/mesosphere/charts/blob/master/staging/docker-registry/values.yaml#L217
we should add affinity in this Values.yaml.

Reply:
Can I create a follow up ticket? This probably needs some more investigation on the cases that we don't really have e2e tests for (ie upgrades, more cp Nodes, etc)